### PR TITLE
[Easy] Add whitespace after comma when re-rendering tuple default value in schema

### PR DESCRIPTION
**Pull request by XuehaiPan**
The default value of `rot90()` in the schema registry is `[0,1]` because we split the function schema by `", "`. There should be no space after `,` in `[0,1]`.

https://github.com/pytorch/pytorch/blob/5c9d5272e43573269427cae5e38307faac3266f6/aten/src/ATen/native/native_functions.yaml#L6120-L6126

Then the the default value is formatted to `(0,1)` in `pyi` files. This PR manually adds an extra whitespace when rerendering the default value to a string.

```python
", ".join(string.split(","))
```

```python
# before
def rot90(input: Tensor, k: _int = 1, dims: _size = (0,1)) -> Tensor: ...
# after
def rot90(input: Tensor, k: _int = 1, dims: _size = (0, 1)) -> Tensor: ...
```

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #129887
* #129873
* #129886
* #129872
* #129420
* #129871
* #129870
* __->__ #129884

cc @ezyang @bhosmer @bdhirsh @kadeng

### CODE REVIEW

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -11423,7 +11423,7 @@ def merge_dicts(*dicts):
 add_docstr(
     torch.rot90,
     r"""
-rot90(input, k=1, dims=[0,1]) -> Tensor
+rot90(input, k=1, dims=(0, 1)) -> Tensor
```
Comment (malfet) [inline torch/_torch_docs.py:11426]: Why change list to a tuple here?

Comment (XuehaiPan) [reply torch/_torch_docs.py:11426]: The default value is a tuple in the stub file. The list is converted to tuple in torchgen.

https://github.com/pytorch/pytorch/blob/8c2c3a03fb87c3568a22362d83b00d82b9fb3db2/torchgen/api/python.py#L276-L282

**Comment by XuehaiPan** [comment]
I found the original PR "Allow whitespace in native function schemas" is non-trivial because there is another schema parser written in C++ with a handcrafted lexer.

https://github.com/pytorch/pytorch/blob/5c9d5272e43573269427cae5e38307faac3266f6/torch/csrc/jit/frontend/function_schema_parser.cpp#L216-L284

`std::stod(...)` cannot parse expressions like `1 / 8` to float `0.125` on the C++ side.

https://github.com/pytorch/pytorch/blob/5c9d5272e43573269427cae5e38307faac3266f6/torch/csrc/jit/frontend/function_schema_parser.cpp#L275-L278

**Comment by ezyang** [comment]
NGL, if this is complicated, I don't really think it's worth it. It's not that big a deal avoiding spaces in the default args.

**Comment by XuehaiPan** [comment]
> NGL, if this is complicated, I don't really think it's worth it. It's not that big a deal avoiding spaces in the default args.

This PR only changes the behavior when we re-render the schema to `pyi` signature.

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**